data/core.yaml

```diff
@@ -944,12 +944,12 @@ en:
     title: Date Range
     start_date:
       description: Start Date
-      tooltip: The start date YYYY-MM-DD format. Negative years OK.
-      placeholder: e.g. 1215-06-15
+      tooltip: The start date in YYYY-MM-DD format, for example 1215-06-15. Use -0001 for the year 2 BCE.
+      placeholder: YYYY-MM-DD
     end_date:
       description: End Date
-      tooltip: The end date YYYY-MM-DD format. Negative years OK.
-      placeholder: e.g. 1969-07-16
+      tooltip: The end date in YYYY-MM-DD format, for example 1969-07-16. Use -0001 for the year 2 BCE.
+      placeholder: YYYY-MM-DD
```

Comment:
I went to fix a typo but wound up swapping the example and format between the tooltip and placeholder. This makes the placeholder consistent with the placeholder in the Start Date and End Date fields of the inspector. I also added another example to illustrate negative years, since some mappers might be unaware of the potential for off-by-one error caused by Year 0.